src/include/crowbar/complex.rb

```diff
@@ -730,7 +730,7 @@ def show_custom_repos_widget
     def InitRPRepos(id)
       # find the initial location now
       if @repos_location.empty?
-        @repos_location = "custom"
+        @repos_location = "local"
```

Comment:
I wouldn't give much time to that, I think we plan to drop this yast module in next cloud release....